readme.md

```diff
@@ -82,6 +90,15 @@ Content-Length: 112
         "day":"MWF",
         "time":"12-1",
         "location":"KEC"
+        },
+        {
+        "cid": 6,
+        "crn": 11112,
+        "courseName": "CS 122",
+        "instructor": "Mr. Test2",
+        "day":"MWF",
+        "time":"12-1",
+        "location":"KEC"
         }
 ]
 ```
```

Comment:
the request is correct, but the response has an incorrect content length and is pretty-printed. the results of an actual request and response copied from your terminal would be more accurate documentation.